uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/components/InputFieldModifierWithFocusChangedLogic.kt

```diff
@@ -0,0 +1,41 @@
+package tech.dojo.pay.uisdk.presentation.components
+
+import androidx.compose.foundation.ScrollState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.onFocusChanged
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlin.math.roundToInt
+
+@Composable
+fun InputFieldModifierWithFocusChangedLogic(
```

Comment:
Same here, this is giving you a warning, use as extension function of Modifier, make it internal so no one from outside is able to access it and rename it to: `Modifier.scrollableInputFieldOnFocusChange(...)`